scipy/stats/_mstats_basic.py

```diff
@@ -2590,13 +2593,16 @@ def moment(a, moment=1, axis=0):
         else:
             return ma.array(np.full(out_shape, np.nan, dtype=dtype))
 
+    if central is None:
+        mean = a.mean(axis, keepdims=True)
+    else:
+        mean = central
```

Comment:
```suggestion
    if center is None:
        center = a.mean(axis, keepdims=True)
```
Using `mean` for this variable suggests that it is always the mean. It is not always the mean (which is the purpose of the PR).

Reply:
> Using `mean` for this variable suggests that it is always the mean. It is not always the mean (which is the purpose of the PR).

Thanks for the suggestion! The code I wrote was probably a functional programming habit leftover :stuck_out_tongue:.